packages/trace-viewer/src/ui/copyToClipboard.css

```diff
@@ -0,0 +1,23 @@
+/*
+  Copyright (c) Microsoft Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+.copy-to-clipboard-text-button {
+  background-color: var(--vscode-editor-inactiveSelectionBackground);
+  border: none;
+  padding: 4px 12px;
+  margin: 8px 8px 8px 0px;
```

Comment:
I think it is strange to put margin into a component. I'd expect the parent component to manage margins and positioning of children.

Reply:
you're right. done in https://github.com/microsoft/playwright/pull/32858/commits/2761ad9d84444950b9579c9115469cc695f62f11